CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# Release v2.1 (2019-05-09)
+
+## Improvements
+* [Datasync][datasync-plugin]
+  - added new option `WithClientLifetimeTTL`. This option defines `put` operation for the lifetime of a client (the TTL is not renewed if the client is closed).
```

Comment:
Sentences should start with capital letter to be consistent with other lists and especially because it ends with dot.